css-basics_social-card/css-basics_button-styles/css-basics_pokemon-page/css/styles.css

```diff
@@ -22,7 +22,9 @@
 
 /* Task 3: To set the line height to 1.5 is good for texts, because of accessibility concerns, but for the headings it seems too much. Let's change this by giving our headings a line height of 1.2: */
 
-/* your solution */
+.headings {
```

Comment:
In this case these elements already have classes assigned to them. You didn't need to create new class names, but simply referred to the elements directly.

h1, h2 {
  line-height: 1.2;
}